src/main/java/seedu/address/model/transaction/TransactionList.java

```diff
@@ -26,7 +26,7 @@ public boolean contains(Transaction toCheck) {
     }
 
     /**
-     * Adds a Transaction to the list.
+     * Adds a Transaction to the list. Multiple identical transactions are allowed.
```

Comment:
According to the user guide, transaction delete will work based on currently displayed list, hence it should be able to distinguish between multiple identical transactions, but i'll take note of this to implement the TransactionDeleteCommand to work by accessing index as well, instead of searching through the list and return the first matching transaction. Thanks!